src/app.js

```diff
@@ -2,48 +2,36 @@ const { app } = require("./support/setupExpress");
 const { query } = require("./support/db");
 const { gameOfThronesEpisodes } = require("./data/gameOfThronesData");
 
-/** 
- @typedef {import('./data/episodeType').Episode} Episode
-*/
 
-//You can delete this once you see the episodes have loaded ok.
-summariseEpisodesToConsole(gameOfThronesEpisodes);
 
-//configure the server's route handlers
 app.get("/", (req, res) => {
-    res.render("pages/index");
+    res.render("pages/index", { 
+        
+    });
 });
 
-app.get("/db-test", async (req, res) => {
-    try {
-        const dbResult = await query("select now()");
-        const rows = dbResult.rows;
-        res.json(rows);
-    } catch (err) {
-        console.error(err);
-        res.status(500).send(
-            "Sorry, an error occurred on the server.  Ask the dev team to check the server logs at time " +
-                new Date(),
-        );
-    }
+app.get("/fullepisodes", (req, res) => { //cannot have spaces between nav names
+    res.render("pages/fullEpisodes", {
```

Comment:
It would be good to be consistent in capitalisation here: the ejs file is `pages/fullEpisodes` but the path is `/fullepisodes`.  
However, capitalisation aside, the name of both is great - a very clear description of what it provides.